config/peerpods/podvm/lib.sh

```diff
@@ -332,9 +332,9 @@ function extract_container_image() {
 
     # Form the skopeo CLI. Add authfile if provided
     if [[ -n "${5}" ]]; then
-        SKOPEO_CLI="skopeo copy --authfile ${auth_json_file}"
+        SKOPEO_CLI="skopeo copy --remove-signatures --authfile ${auth_json_file}"
```

Comment:
nit:
You can add a comment that "oci image destination supports signatures" hence the signatures if any are removed while copying